html-relative-links/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="index.html">Home Page</a></li>
+    <li><a href="about-me/about-me.html">About Me</a></li>
+    <li><a href="projects/projects.html">Projects</a></li>
+    <li><a href="contact-us/contact-us.html">Contact Us</a></li>
+  </ul>
+  <p>Hello Homepage!</p>
+  <img src="https://i0.heartyhosting.com/www.surfer.com/wp-content/uploads/2020/10/SR1020_PROSURF_u2329430.jpg?resize=1920%2C1080&ssl=1" alt="Guy surfing big wave">
```

Comment:
Please use a relative url to the images provided in the starter files.

Reply:
<img width="1312" alt="Screen Shot 2022-01-25 at 11 13 32 AM" src="https://user-images.githubusercontent.com/96929704/151043939-2a546a31-6e16-4e12-b99b-5ca4196be7f8.png">